hack/tools.go

```diff
@@ -19,7 +19,6 @@ limitations under the License.
 package tools
 
 import (
-	_ "k8s.io/code-generator"
```

Comment:
@kron4eg It seems like this change breaks the verify codegen job.

Reply:
yeah, reverted it